docs/user/gen-docs/kyma_completion.md

```diff
@@ -0,0 +1,37 @@
+# kyma completion
+
+Generate the autocompletion script for the specified shell.
+
+## Synopsis
+
+Generate the autocompletion script for kyma for the specified shell.
```

Comment:
```suggestion
Generate the autocompletion script for Kyma for the specified shell.
```

Reply:
this is function from the library and we can't change it